PocketApp.swift

```diff
@@ -4,24 +4,34 @@
 
 import PocketKit
 import SwiftUI
+import SwiftData
 
 @main
 struct PocketApp: App {
     @UIApplicationDelegateAdaptor private var delegate: PocketAppDelegate
 
     @Environment(\.scenePhase)
-    var scenePhase
+    private var scenePhase
 
-    @StateObject private var rootViewModel = RootViewModel()
+    @StateObject private var rootViewModel: RootViewModel
+
+    private let modelContainer: ModelContainer
+
+    init() {
+        _rootViewModel = StateObject(wrappedValue: RootViewModel())
+        modelContainer = DataController().makeModelContainer()
```

Comment:
this was moved here from `Services` so that the container is now available to all the app.